source/fundamentals/connections/tls.txt

```diff
@@ -0,0 +1,212 @@
+   default.
+
+For a full list of client options, see :ref:`golang-connection-options`.
+
+.. _golang-configure-tls-certificates:
+
+Configure Certificates
+----------------------
+
+To successfully initiate a TLS request, your application must present 
+cryptographic certificates to prove its identity. Your application's
+certificates must be stored as PEM files to enable TLS when connecting.
+
+.. important::
+
+   For production use, we recommend that your MongoDB deployment use valid
+   certificates generated and signed by the same certificate authority.
+   For testing, your deployment can use self-signed certificates.
+
+The following list describes the components that your client must
+present to establish a TLS-enabled connection:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - TLS Component
+     - Description
+
+   * - Certificate Authority (CA)
+     - One or more certificate authorities to
+       trust when making a TLS connection.
+
+   * - Client Certificate
+     - A digital certificate that allows the server to verify the identity
+       of your application to establish an encrypted network connection.
+
+   * - Certificate Key
+     - The client certificate private key file. This key is often
+       included within the certificate file itself.
+
+   * - Passphrase
+     - The password to decrypt the private client key if it is encrypted.
+
+.. _golang-client-tls-connect:
+
+Reference Certificates in a Client
+----------------------------------
+
+You must reference your certificates in your ``ClientOptions``
+object so that the server can validate them before the client connects.
+We recommend that you set the ``TLSConfig`` field of your
+``ClientOptions`` instance to a ``Config`` struct to configure your
+TLS connection. ``Config`` structs are native to Go and allow you to keep
+all your TLS options in a single reusable object.
+
+To create a ``Config`` instance, import the ``crypto/tls`` and
+``crypto/x509`` packages. Next, create a ``Config`` struct instance and
+set the relevant struct fields for your configuration.
+
+Within your ``Config`` instance, you can set optional
+fields to configure TLS on your connection. For **testing purposes**,
+you can set the ``InsecureSkipVerify`` field to ``true``.
+   
+.. warning::
+
+   Setting the ``InsecureSkipVerify`` field to ``true`` disables
+   both certificate and hostname validation.
+   
+   Specifying this option in a production environment makes
+   your application insecure and potentially
+   vulnerable to expired certificates and foreign processes posing
+   as valid client instances.
+
+To learn more about the ``Config`` struct, see the `tls.Config API
+documentation <https://pkg.go.dev/crypto/tls#Config>`__.
+
+.. _golang-tls-config-full-example:
+
+Example
+~~~~~~~
+
+This example performs the following actions to create a ``Config``
+instance and a ``Client`` instance that includes the TLS options:
```

Comment:
s: "the TLS options" stands out to me. could it be changed to "instance enabled for TLS" or "the previous TLS options" (if that's true)?